src/main/CommandLine.cpp

```diff
@@ -1537,8 +1537,19 @@ handleCommandLine(int argc, char* const* argv)
     }
     catch (std::exception& e)
     {
-        LOG_FATAL(DEFAULT_LOG, "Got an exception: {}", e.what());
-        LOG_FATAL(DEFAULT_LOG, "{}", REPORT_INTERNAL_BUG);
+        std::string errMsg = fmt::format("Got an exception: {}", e.what());
+        if (Logging::loggingInitialized())
```

Comment:
the reason I had to do is this because when we call `setLoggingToFile` and it fails, logger is left in a bad state, so any default log afterwards wouldn't work. I've updated the code to instead reset and re-initialize the logger in `setLoggingToFile`, so that it's in valid state before throwing.